docs/charts/line.md

```diff
@@ -41,37 +41,86 @@ var myLineChart = new Chart(ctx, {
+| Name | Type | [Scriptable](../general/options.md#scriptable-options) | [Indexable](../general/options.md#indexable-options) |  Default
+| ---- | ---- | :----: | :----: | ----
+| [`backgroundColor`](#line-styling) | [`Color`](../general/colors.md) | - | - | `'rgba(0,0,0,0.1)'`
+| [`borderCapStyle`](#line-styling) | `String` | - | - | `'butt'`
+| [`borderColor`](#line-styling) | [`Color`](../general/colors.md) | - | - | `'rgba(0,0,0,0.1)'`
+| [`borderDash`](#line-styling) | `Number[]` | - | - | `[]`
+| [`borderDashOffset`](#line-styling) | `Number` | - | - | `0`
+| [`borderJoinStyle`](#line-styling) | `String` | - | - | `'miter'`
+| [`borderWidth`](#line-styling) | `Number` | - | - | `0`
+| [`cubicInterpolationMode`](#cubicInterpolationMode) | `String` | - | - | `''`
+| [`fill`](#line-styling) | `Boolean/String` | - | - | `true`
+| [`label`](#general) | `String` | - | - | `''`
+| [`lineTension`](#line-styling) | `Number` | - | - | `0.4`
+| [`pointBackgroundColor`](#point-styling) | `Color` | Yes | Yes | `'rgba(0,0,0,0.1)'`
+| [`pointBorderColor`](#point-styling) | `Color` | Yes | Yes | `'rgba(0,0,0,0.1)'`
+| [`pointBorderWidth`](#point-styling) | `Number` | Yes | Yes | `1`
+| [`pointHitRadius`](#point-styling) | `Number` | Yes | Yes | `1`
+| [`pointHoverBackgroundColor`](#interactions) | `Color` | Yes | Yes | `undefined`
+| [`pointHoverBorderColor`](#interactions) | `Color` | Yes | Yes | `undefined`
+| [`pointHoverBorderWidth`](#interactions) | `Number` | Yes | Yes | `undefined`
+| [`pointHoverRadius`](#interactions) | `Number` | Yes | Yes | `undefined`
+| [`pointRadius`](#point-styling) | `Number` | Yes | Yes | `3`
+| [`pointRotation`](#point-styling) | `Number` | Yes | Yes | `1`
+| [`pointStyle`](#point-styling) | `String/Image` | Yes | Yes | `'circle'`
+| [`showLine`](#general) | `Boolean` | - | - | `undefined`
+| [`spanGaps`](#general) | `Boolean` | - | - | `false`
+| [`steppedLine`](#stepped-line) | `Boolean/String` | - | - | `false`
+| [`xAxisID`](#general) | `String` | - | - | first x axis
+| [`yAxisID`](#general) | `String` | - | - | first y axis
+
+### General
+
+| Name | Description
+| ---- | ----
+| `label` | The label for the dataset which appears in the legend and tooltips.
+| `xAxisID` | The ID of the x axis to plot this dataset on.
+| `yAxisID` | The ID of the y axis to plot this dataset on.
+
+### Point Styling
+
+The style of each point can be controlled with the following properties:
+
+| Name | Description
+| ---- | ----
+| `pointBackgroundColor` | The fill color for points.
+| `pointBorderColor` | The border color for points.
+| `pointBorderWidth` | The width of the point border in pixels.
+| `pointHitRadius` | The pixel size of the non-displayed point that reacts to mouse events.
+| `pointRadius` | The radius of the point shape. If set to 0, the point is not rendered.
+| `pointRotation` | The rotation of the point in degrees.
+| `pointStyle` | Style of the point. [more...](../configuration/elements#point-styles)
+
```

Comment:
To be consistent [with other charts](/chartjs/Chart.js/blob/master/docs/charts/bubble.md#styling), should we add something like:

```
All these values, if `undefined`, fallback first to the dataset options then to the associated [`elements.point.*`](../configuration/elements.md#point-configuration) options. 
```

Reply:
Added